hack/generate-rook-csv.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+set -e
+
+#############
+# VARIABLES #
+#############
+
+operator_sdk="${OPERATOR_SDK:-operator-sdk}"
+yq="${YQv3:-yq}"
+PLATFORM=$(go env GOARCH)
+
+YQ_CMD_DELETE=("$yq" delete -i)
+YQ_CMD_MERGE_OVERWRITE=("$yq" merge --inplace --overwrite --prettyPrint)
+YQ_CMD_MERGE=("$yq" merge --arrays=append --inplace)
+YQ_CMD_WRITE=("$yq" write --inplace -P)
+CSV_FILE_NAME="deploy/csv-templates/crds/rook/manifests/rook-ceph.clusterserviceversion.yaml"
+CEPH_EXTERNAL_SCRIPT_FILE="https://raw.githubusercontent.com/rook/rook/master/deploy/examples/create-external-cluster-resources.py"
+wget --no-verbose $CEPH_EXTERNAL_SCRIPT_FILE
+ASSEMBLE_FILE_COMMON="deploy/csv-templates/rook_olm_files/metadata-common.yaml"
+ASSEMBLE_FILE_OCP="deploy/csv-templates/rook_olm_files/metadata-ocp.yaml"
+#############
+# FUNCTIONS #
+#############
+
+function generate_csv() {
+    # If you want to use specific branch, tags or commit has refer to this doc https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#remote-directories
+    kubectl kustomize https://github.com/rook/rook//deploy/examples | "$operator_sdk" generate bundle --package="rook-ceph" --output-dir="deploy/csv-templates/crds/rook/" --extra-service-accounts=rook-ceph-system,rook-csi-rbd-provisioner-sa,rook-csi-rbd-plugin-sa,rook-csi-cephfs-provisioner-sa,rook-csi-nfs-provisioner-sa,rook-csi-nfs-plugin-sa,rook-csi-cephfs-plugin-sa,rook-ceph-system,rook-ceph-rgw,rook-ceph-purge-osd,rook-ceph-osd,rook-ceph-mgr,rook-ceph-cmd-reporter
+
+    directory="deploy/csv-templates/crds/rook/manifests"
```

Comment:
Can we move this to config/rook

Reply:
since we store rook csv crd in `deploy/csv-templates` I have put it under that. Let me know if you want to put it under `config/rook`. Thanks.